test/services/migrationService.test.ts

```diff
@@ -1,8 +1,12 @@
+import { mocked } from 'ts-jest/utils'
 import MigrationService, { Flag } from '../../server/services/migrationService'
 import { delius } from '../../server/config'
+import { DeliusClient, StaffDetails } from '../../server/data/deliusClient'
+
+jest.mock('../../server/data/deliusClient')
 
 describe('MigrationService', () => {
-  let deliusClient
+  let deliusClient: DeliusClient
```

Comment:
Not sure but think this could be simplified by declaring as: `deliusClient: jest.Mocked<DeliusClient>` ? 
then you would only need to use `mocked` once?

`deliusClient = mocked(new DeliusClient(undefined))`

rather than wrapping everytime you want to do an assertion..